app/helpers/countries_helper.rb

```diff
@@ -1,29 +1,41 @@
 module CountriesHelper
-  def if_country_get_flag yml_key
-    country = I18n.exists?("countries.#{yml_key}.image_flag")
-    flag = FileTest.exist?("app/assets/images/#{t("countries.#{yml_key}.image_flag")}")
+  def if_country_get_flag(country_name)
+    name = country_name.downcase.gsub(' ', '-')
+    flag = FileTest.exist?("app/assets/images/flags/#{name}.svg")
 
-    country && flag ? image_tag(I18n.t("countries.#{yml_key}.image_flag"), alt: '', class: 'header__icon') : nil
+    flag ? image_tag("flags/#{name}.svg", alt: '', class: 'header__icon') : nil
   end
 
   def nav_tertiary_countries
     I18n.t('countries.nav_sticky').to_json
   end
 
   def dataset_status(dataset)
-    dataset_status = @country.occurrences[dataset[:id]]
+    @country.occurrences[dataset[:id]]
   end
 
   def dataset_status_title(dataset)
-    dataset_status_title = I18n.t("shared.proportion_protected.title_#{dataset_status(dataset)}")
+    status = dataset_status(dataset) == 'present-but-unknown' ? 'data_deficient' : dataset_status(dataset)
```

Comment:
Only a small thing - if I'm calling a method twice, I usually assign it to a variable first:

```
status = dataset_status(dataset)
status_key = status == 'present-but-unknown' ? 'data_deficient' : status
```

Because this is such a simple method, so there aren't performance implications, up to you which you prefer :)